Source/ReactiveCore/MoyaError.swift

```diff
@@ -10,11 +20,23 @@ public enum MoyaErrorCode: Int {
     case Data
 }
 
-public enum MoyaError: ErrorType {
-    case ImageMapping(MoyaResponse)
-    case JSONMapping(MoyaResponse)
-    case StringMapping(MoyaResponse)
-    case StatusCode(MoyaResponse)
-    case Data(MoyaResponse)
-    case Underlying(ErrorType)
+internal extension MoyaError {
+    
+    // Used to convert MoyaError to NSError for RACSignal
+    var nsError: NSError {
```

Comment:
Would love to but I can't.
![screen shot 2015-11-12 at 3 04 24 pm](https://cloud.githubusercontent.com/assets/296718/11129669/b61a6a2e-894e-11e5-9247-e4f887b6b47b.png)